scripts/screener/screener.iefiles.ts

```diff
@@ -0,0 +1,82 @@
+const defaultExamples = [
+  'Alert',
+  'Accordion',
+  'Animation',
+  'Attachment',
+  'Avatar',
+  'Box',
+  'Button',
+  'Card',
+  'Carousel',
+  'Checkbox',
+  'Dialog',
+  'Divider',
+  'Embed',
+  'Form',
+  'Header',
+  'HierarchicalTree',
+  'Image',
+  'Input',
+  'ItemLayout',
+  'Label',
+  'List',
+  'Loader',
+  'Menu',
+  'MenuButton',
+  'Popup',
+  'Portal',
+  'Provider',
+  'RadioGroup',
+  'Reaction',
+  'Ref',
+  'Segment',
+  'Slider',
+  'SplitButton',
+  'Status',
+  'SvgIcon',
+  'Table',
+  'Text',
+  'TextArea',
+  'Toolbar',
+  'Tooltip',
+  'Tree',
+  'Video',
+].map(example => new RegExp(`^${example}Example(.shorthand)?.tsx$`));
+
+const specificExamples = [
+  'AttachmentActionableExample',
+  'AvatarExampleSize',
+  'ButtonExampleContentAndIcon',
+  'CardExampleSize',
+  'CardExampleCentered',
+  'CardExampleHorizontal',
+  'CardExampleFocusableChildrenGrid',
+  'CardExampleFocusableChildrenGrid',
+  'CarouselThumbnailsExample',
+  'CarouselPaginationExample',
+  'Chat',
+  'CheckboxExampleChecked',
+  'DialogExampleBackdrop',
+  'DividerExampleContent',
+  'DividerExampleVertical',
+  'Dropdown',
+  'Flex',
+  'Grid',
+  'InputExampleIcon',
+  'ItemLayoutExampleMedia',
+  'LabelExampleContentCustomization',
+  'Layout',
+  'ListExampleMedia',
+  'ListExampleHorizontal',
+  'MenuExampleVertical',
+  'MenuButtonExampleContextMenu',
+  'PopupCustomTargetExample',
+  'SplitButtonPositioningExampleShorthand',
+  'TableExampleNavigable',
+  'ToolbarExampleMenuWithSubmenu',
+  'TooltipExamplePointing',
+  'TreeInitiallyOpenExample',
+  'TreeMultiselectExample',
```

Comment:
Can we configure this under `.steps.ts` files instead?

Reply:
Not all examples have `.steps` files and the way the screener config works we need a list of files to match, so the way I see it could work under the example files would be if we change files to be tested to something like `TreeMultiselectExample.shorthand.tsx` to `TreeMultiselectExample.shorthand.ie.tsx` and then have the Regex to match it. 

Or do you see other way?